config/kdump.py

```diff
@@ -1,44 +1,69 @@
-import os
+import sys
```

Comment:
>sys [](http://example.com/codeflow?start=7&length=3)

Since you are fixing a bug. Please add a test case which could fail previous code, but pass with new code.

Reply:
Added the unit test cases.